settings.gradle

```diff
@@ -24,39 +24,49 @@ pluginManagement {
 
 rootProject.name = 'bookkeeper'
 
-include 'bookkeeper-benchmark',
+include('circe-checksum',
+        'circe-checksum:src:main:circe',
+        'cpu-affinity',
+        'cpu-affinity:src:main:affinity',
         'bookkeeper-common',
-        'bookkeeper-common-allocator',
         'bookkeeper-http:http-server',
         'bookkeeper-http:vertx-http-server',
-        'bookkeeper-proto',
         'bookkeeper-server',
+        'bookkeeper-common-allocator',
+        'bookkeeper-proto',
         'bookkeeper-stats',
+        'tests:integration-tests-base',
+        'tests:integration-tests-utils',
+        'tests:integration-tests-topologies',
+        'tests:integration:cluster',
+        'tests:integration:smoke',
+        'tests:integration:standalone',
```

Comment:
Can we keep this list sorted. That will make it easier to resolve conflicts

Reply:
Yeah.